src/test/groovy/com/palantir/gradle/docker/DockerComposePluginTests.groovy

```diff
@@ -65,8 +65,8 @@ class DockerComposePluginTests extends AbstractPluginTest {
         when:
         BuildResult buildResult = with('generateDockerCompose', '--stacktrace').buildAndFail()
         then:
-        buildResult.standardError.contains("Failed to resolve Docker dependencies declared in")
-        buildResult.standardError.contains("{{foo:bar}}")
+        buildResult.output.contains("Failed to resolve Docker dependencies declared in")
```

Comment:
I understand that the plugin will still work with <2.10, but the stderr/stdout semantics changed for 2.10, hence this change?